src/Features/VisualBasic/Portable/Completion/CompletionProviders/TypeImportCompletionProvider.vb

```diff
@@ -0,0 +1,41 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.Completion.Providers
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.Shared.Extensions.ContextQuery
+Imports Microsoft.CodeAnalysis.Text
+Imports Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports System.Collections.Immutable
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
+
+    Friend NotInheritable Class TypeImportCompletionProvider
+        Inherits AbstractTypeImportCompletionProvider
+
+        Friend Overrides Function IsInsertionTrigger(text As SourceText, characterPosition As Integer, options As OptionSet) As Boolean
+            Return CompletionUtilities.IsDefaultTriggerCharacterOrParen(text, characterPosition, options)
+        End Function
+
+        Protected Overrides Async Function CreateContextAsync(document As Document, position As Integer, cancellationToken As CancellationToken) As Task(Of SyntaxContext)
+            ' Need regular semantic model because we will use it to get imported namepsace symbols.
+            Dim semanticModel = Await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(False)
```

Comment:
idon't understand the comment.  what's wrong with using GetSemanticModelForSpanAsync here? If there is a problem, the comment needs to be fleshed out better to explain what the issue is.  thanks!

Reply:
Because we later try to get imported namespace symbols, which is outside of the span, and end up with this

![image](https://user-images.githubusercontent.com/788783/55858696-36092f80-5b25-11e9-9c9d-e3db1e130bdc.png)

Reply:
ah i see.  can you expand the comment just slightly?

Reply:
done